packages/devtools_app/test/vm_developer/object_inspector/object_viewport_test.dart

```diff
@@ -44,6 +49,8 @@ void main() {
 
     testObjectInspectorViewController = TestObjectInspectorViewController();
 
+    // MockClassObject setUp
```

Comment:
might be work breaking this test up into multiple groups that test the different object types so that this setup isn't so long.

Reply:
+1

Reply:
Done